packages/atomic/src/components/atomic-refine-modal/atomic-refine-modal.tsx

```diff
@@ -177,9 +177,9 @@ export class AtomicRefineModal implements InitializableComponent {
     }
 
     return [
-      <div part="section-title" class="text-2xl font-bold truncate mb-3 mt-8">
+      <h1 part="section-title" class="text-2xl font-bold truncate mb-3 mt-8">
```

Comment:
Is this really a good idea to use headings here ? Don't we risk messing up the existing heading in clients website ? 🤔

Reply:
at first, I wanted to introduce a prop for headings, but this is a modal, therefor screen readers recognize its content as being the only content of the entire website, while it's open. It felt appropriate to give it level-1 headings.

Reply:
Alright ! I think it's logical indeed, with the focus trap.